src/java.base/share/man/java.1

```diff
@@ -3853,7 +3853,7 @@ This option was deprecated in JDK 16 by \f[B]JEP 396\f[R]
 403\f[R] [https://openjdk.org/jeps/403].
 .SH REMOVED JAVA OPTIONS
 .PP
-These \f[V]java\f[R] options have been removed in JDK 23 and using them
+These \f[V]java\f[R] options have been removed in JDK 24 and using them
```

Comment:
This is incorrect. You can't just change 23 to 24 here as the actual set of flags listed below will be will be different.

Reply:
David, my understanding is that you agree that this is okay (per our out-of-band, follow-up discussion).